app/scripts/controllers/login-controller.js

```diff
@@ -59,9 +59,9 @@ angular.module('stellarClient').controller('LoginCtrl', function($rootScope, $sc
   }
 
   FlashMessages.add({
-    title: 'The Stellar network has upgraded!',
-    info: 'If you\'ve already upgraded, view your lumens balance at https://www.stellar.org/account-viewer/. To upgrade, or to see what you still have in addition to lumens, log in below.',
-    type: 'info',
+    title: 'The Account Upgrade service is temporarily unavavilable',
```

Comment:
💡  Typo here:

```suggestion
    title: 'The Account Upgrade service is temporarily unavailable',
```

Reply:
Oh great catch – will send a PR shortly to address this since the page is live.

Reply:
Committed directly to master in 60f0b8c